res/css/views/elements/_ImageView.scss

```diff
@@ -38,12 +38,23 @@ $button-gap: 24px;
     flex-shrink: 0;
 }
 
+@keyframes mx_ImageView_panel_keyframes {
```

Comment:
We have `_animations.scss`, could you move it there instead? I think it'll be much better to keep all the animations stuff in one file, so that we can normalize it in all in the future instead of reinventing the wheel all the time and having different animations in different parts of the app.
Full path: `res/css/_animations.scss`